packages/mobile/src/send/SendAmount.tsx

```diff
@@ -157,9 +157,15 @@ export class SendAmount extends React.Component<Props, State> {
   fetchLatestAddressesAndValidate = () => {
     const { recipient } = this.props
 
-    if (recipient.e164PhoneNumber) {
-      this.props.fetchAddressesAndValidate(recipient.e164PhoneNumber)
+    if (recipient.kind === RecipientKind.QrCode) {
+      return
+    }
+
+    if (!recipient.e164PhoneNumber) {
+      throw Error('Recipient phone number is required if not sending via QR Code')
```

Comment:
This will throw if the recipient type is `RecipientKind.Address` right? That's what we use if the use types/pastes an address in to the send field. You can fix this by adding an `||` case for it above :)